### PR TITLE
Exclude Cluster API namespaces from Velero backups

### PR DESCRIPTION
**Pull request by anders-elastisys**
> [!warning]
> **This is a public repository, ensure not to disclose:**
>
> - [x] personal data beyond what is necessary for interacting with this pull request, nor
> - [x] business confidential information, such as customer names.

### What kind of PR is this?

**Required**: Mark one of the following that is applicable:

- [ ] kind/feature       
- [x] kind/improvement   
- [ ] kind/deprecation   
- [ ] kind/documentation 
- [ ] kind/clean-up      
- [ ] kind/bug           
- [ ] kind/other         

*Optional*: Mark one or more of the following that are applicable:

> [!important]
> Breaking changes should be marked `kind/admin-change` or `kind/dev-change` depending on type
> Critical security fixes should be marked with `kind/security`

- [ ] kind/admin-change 
- [ ] kind/dev-change   
- [ ] kind/security     
- [ ] [kind/adr]()      

### What does this PR do / why do we need this PR?

When doing a Velero restore in both Workload and Management clusters I noticed that Cluster API namespaces got included and restored. As we exclude other system namespaces we should exclude these as well in Veleros daily backups. This PR addresses this.

- Fixes #

#### Information to reviewers

I initially wanted to put capi specific namespaces for `wc` in the [k8sInstaller capi config](https://github.com/elastisys/compliantkubernetes-apps/tree/main/config/k8s-installers/capi) but realized that we don't merge arrays when we init config, so I added them to the complete list of excludedNamespaces instead.

#### Checklist

- [x] Proper commit message prefix on all commits
  
- Change checks:
  - [x] The change is transparent
  - [ ] The change is disruptive
  - [ ] The change requires no migration steps
  - [ ] The change requires migration steps
  - [ ] The change upgrades CRDs
  - [ ] The change updates the config *and* the schema
- Documentation checks:
  - [ ] The [public documentation](https://github.com/elastisys/compliantkubernetes) required no updates
  - [ ] The [public documentation](https://github.com/elastisys/compliantkubernetes) required an update - [link to change](set-me\)
- Metrics checks:
  - [ ] The metrics are still exposed and present in Grafana after the change
  - [ ] The metrics names didn't change (Grafana dashboards and Prometheus alerts are not affected)
  - [ ] The metrics names did change (Grafana dashboards and Prometheus alerts were fixed)
- Logs checks:
  - [ ] The logs do not show any errors after the change
- Pod Security Policy checks:
  - [ ] Any changed pod is covered by Pod Security Admission
  - [ ] Any changed pod is covered by Gatekeeper Pod Security Policies
  - [ ] The change does not cause any pods to be blocked by Pod Security Admission or Policies
- Network Policy checks:
  - [ ] Any changed pod is covered by Network Policies
  - [ ] The change does not cause any dropped packets in the `NetworkPolicy Dashboard`
- Audit checks:
  - [ ] The change does not cause any unnecessary Kubernetes audit events
  - [ ] The change requires changes to Kubernetes audit policy
- Falco checks:
  - [ ] The change does not cause any alerts to be generated by Falco
- Bug checks:
  - [ ] The bug fix is covered by regression tests

### CODE REVIEW

**Comment by aarnq** [comment]
Don't we want to backup (at least) capi-clusters in the very corner case where the management cluster fails and the workload cluster continues to function?
Because then we can actually restore management function for the workload cluster.

**Review by aarnq — APPROVED** [review]

**Review by Zash — APPROVED** [review]

File: helmfile.d/values/velero-sc.yaml.gotmpl [hunk 1]
```diff
@@ -143,6 +143,8 @@ schedules:
       labelSelector:
         matchLabels:
           velero: backup
+      includedNamespaces:
+        - monitoring
```
Comment (OlleLarsson) [inline helmfile.d/values/velero-sc.yaml.gotmpl:147]: Can this be a separate commit as it looks like a separate fix from the exclusion of the capi namespaces?

Comment (anders-elastisys) [reply helmfile.d/values/velero-sc.yaml.gotmpl:147]: I guess since it excludes all namespaces but `monitoring`, not _just_ capi namespaces, I can have it as a separate commit

**Review by OlleLarsson — APPROVED** [review]
